### ISSUE TITLE
A11y : Rethink Tooltip in InlineField to be accessible to screen readers

### ISSUE THREAD

**Issue by yaelleC** [grafana/grafana#47138]
**Steps to reproduce**:

Anywhere a tooltip is used with `InlineField`:

The tooltip is associated with an info icon. #47137 made the icon keyboard accessible but the tooltip text would not be read to screen reader users.

**Actual Result**:

![161065847-0272f6b1-f9d5-44cf-aae6-c66047edce5a](https://user-images.githubusercontent.com/42030685/161066863-f6c294e4-ffdb-4155-ab4c-e1c6881ed34a.png)

**Expected Result**

The `Tooltip` component could be modified:
The tooltip text should be associated with the input rather than the info icon. That way the help text could be read by screen readers (using `role="tooltip"` and `aria-describedby` props?).

**Relevant WCAG Criteria:** 
[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Comment by JoaoSilvaGrafana** [comment]
Closing in favor of https://github.com/grafana/grafana/issues/57207
